### PR TITLE
PBL-36691: Extend pybluetooth to create a connection / disconnect

### PR DESCRIPTION
**Pull request by martijnthe**

### CODE REVIEW

**Comment by martijnthe** [comment]
@chrisc11 @tyhoff @cpattenden @dethredic

File: examples/connect.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+LOG.addHandler(lsh)
+
+b = BTStack()
+b.start()
+
+u = BTStackSynchronousUtils(b)
+
+# Attempt to connect to a device by address:
+try:
+    connection = u.connect(Address("c9:ea:a5:b8:c8:10"), timeout=5.0)
```
Comment (chrisc11) [inline examples/connect.py:27]: I guess just a hardcoded address for now?

**Comment by chrisc11** [comment]
seems reasonable to me, cool stuff! 👍

File: examples/connect.py (hunk 1 above)
Comment (martijnthe) [inline examples/connect.py:27]: Yeah, this file is just an example of how to use the API.

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
--e git+https://github.com/pebble/scapy.git@pebble-master#egg=scapy
+scapy==2.3.2.dev0
 dnet==1.12
```
Comment (chrisc11) [inline requirements.txt:2]: Where did you find this pip library? I had to do "brew install libdnet"

Comment (martijnthe) [inline requirements.txt:2]: Oh, `scapy==2.3.2.dev0` should be on our internal pypi?

```
$ pip install scapy==2.3.2.dev0 --no-cache-dir
Collecting scapy==2.3.2.dev0
  Downloading http://pypi.hq.getpebble.com/packages/scapy-2.3.2-dev.tar.gz (1.2MB)
    100% |████████████████████████████████| 1.2MB 15.0MB/s
Installing collected packages: scapy
  Running setup.py install for scapy
Successfully installed scapy-2.3.2.dev0
```

**Comment by martijnthe** [comment]
Closing because merged manually.

**Pull request closed without merging**